aiida/orm/importexport.py

```diff
@@ -1750,6 +1752,14 @@ def export_tree(what, folder,allowed_licenses=None, forbidden_licenses=None,
         else:
             raise ValueError("I was given {}, which is not a DbNode or DbGroup instance".format(entry))
 
+    # for group in given_groups:
```

Comment:
Ah that is a mistake and probably was not caught by the test I added for the reason you pointed out there. I will write another test that just adds a single disconnected node in the group, to make sure that that is the only reason it can be added